core/src/main/java/org/axonframework/queryhandling/UpdateHandler.java

```diff
@@ -16,5 +16,38 @@
 
 package org.axonframework.queryhandling;
 
+/**
+ * Defines update handler callbacks for subscription query.
+ *
+ * @param <I> the type of initial result
+ * @param <U> the type of incremental updates
+ * @since 3.3
```

Comment:
I'd add an `@author` tag here with your name on it :-)

Reply:
I think that Allard created this interface, will put his name ;)

Reply:
But you actually implemented it?!
Kidding though, I'm fine either way :p